### ISSUE TITLE
No win_amd64 python wheels for non pre-release releases on PyPI

### ISSUE THREAD

**Issue by Deluvi** [mozilla/DeepSpeech#2390]
**TL;DR**: the 64 bits Windows Python wheels are not uploaded on PyPI on releases (e.g 0.5.1) while it is uploaded for pre-releases (e.g 0.6.0 alphas).

- **OS Platform and Distribution**: Windows 10 Home (64 bits) Version 1903
- **Python version**: Python 3.7.0

Hi! I was attempting to run the example on README.md on my Windows computer.

Unfortunately, I got the following error:

```
deepspeech --model deepspeech-0.5.1-models/output_graph.pbmm --alphabet deepspeech-0.5.1-models/alphabet.txt --lm deepspeech-0.5.1-models/lm.binary --trie deepspeech-0.5.1-models/trie --audio test.wav
Loading model from file deepspeech-0.5.1-models/output_graph.pbmm
TensorFlow: v1.14.0-16-g3b4ce374f5
DeepSpeech: v0.6.0-alpha.8-0-gf0e9541
...
Specified model file version (0) is incompatible with minimum version supported by this client (2).
...
```

After some verifications, I found this:

```
pip install deepspeech
Collecting deepspeech
  Downloading https://files.pythonhosted.org/packages/da/fa/457804ae9f2b972961732b3c3bc4efa6c97cedce04a072597d74fa1f03a4/deepspeech-0.6.0a8-cp37-cp37m-win_amd64.whl (7.0MB)
...
deepspeech --version
TensorFlow: v1.14.0-16-g3b4ce374f5
DeepSpeech: v0.6.0-alpha.8-0-gf0e9541
```

Indeed, the `deepspeech` installed by `pip` was a 0.6.0 prerelease (`v0.6.0-alpha.8-0-gf0e9541`) and the model I am using is for the version 0.5.1.

I tried to install the correct version by forcing the version:

```
pip install deepspeech==0.5.1
Collecting deepspeech==0.5.1
  ERROR: Could not find a version that satisfies the requirement deepspeech==0.5.1 (from versions: 0.5.0a5, 0.5.0a6, 0.5.0a7, 0.5.0a8, 0.6.0a5, 0.6.0a6, 0.6.0a7, 0.6.0a8)
  ERROR: No matching distribution found for deepspeech==0.5.1
```

**It seems that the Windows wheels are published on PyPI only on certain pre-releases.** For example, you can see on [version 0.5.1](https://pypi.org/project/deepspeech/0.5.1/#files) that there are no Windows wheels, as opposed to [version 0.6.0a8](https://pypi.org/project/deepspeech/0.6.0a8/#files) where the `deepspeech-0.6.0a8-cp36-cp36m-win_amd64.whl` Windows wheel is present.

I could, however, install deepspeech using [the wheel in the Github release page](https://github.com/mozilla/DeepSpeech/releases/download/v0.5.1/deepspeech-0.5.1-cp37-cp37m-win_amd64.whl):

```
pip uninstall deepspeech
...
pip install deepspeech-0.5.1-cp37-cp37m-win_amd64.whl
...
deepspeech --version
TensorFlow: v1.13.1-10-g3e0cc5374d
DeepSpeech: v0.5.1-0-g4b29b78
```

And now it works perfectly:

```
deepspeech --model deepspeech-0.5.1-models/output_graph.pbmm --alphabet deepspeech-0.5.1-models/alphabet.txt --lm deepspeech-0.5.1-models/lm.binary --trie deepspeech-0.5.1-models/trie --audio test.wav
...
Running inference.
hello world
Inference took 3.733s for 2.735s audio file.
```

**Comment by lissyx** [comment]
Thanks for noticing. It seems some part of the upload indeed silently failed back then. I've retriggered task upload, but there's still some 400 error blocking the upload. v0.5.1 is done, we're not going to have the opportunity to fix that.

As you mentionned, the packages are still available from the github page, and it's working. And v0.6 alphas are uploading correctly, so this should be properly dealt with at this time.
